app/models/user.rb

```diff
@@ -15,12 +15,20 @@
 #
 
 class User < ActiveRecord::Base
+  USER_ROLES = ['User', 'Admin']
+
   has_many :application_users
   has_many :applications, through: :application_users
 
   before_create :generate_api_token
 
+  validates_inclusion_of :role, in: USER_ROLES
```

Comment:
Prefer the new style validations `validates :column, inclusion: value` over `validates_inclusion_of`.